bin/build-release-ccd-definition.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+set -eu
+
+build_dir=$(realpath $(dirname ${0})/../build/release-config)
+prod_config_dir=$build_dir/definitions
+dev_config_dir=$(dirname ${0})/../ccd-definition
+prod_definition_filename=CCD_FPL_PROD_V1_0.xlsx
+
+#create build folder, copy development config and remove the users
+mkdir -p $prod_config_dir
+cp -a $dev_config_dir/. $prod_config_dir
+rm $prod_config_dir/UserProfile.json
+
+#build the ccd definition file
+touch ${build_dir}/${prod_definition_filename}
+export CCD_DEF_CASE_SERVICE_BASE_URL=http://fpl-case-service-prod.service.core-compute-prod.internal
+docker run --rm --name json2xlsx \
```

Comment:
To remove duplicates we could parametrise `kubernetes/configurer/utils/fpl-process-definition.sh` script so that it accepts an optional argument pointing to root directory, behaving as is if value is not provided. Having this we could just execute the script with `$build_dir` as an argument.